ql/instruments/makeois.hpp

```diff
@@ -66,7 +66,13 @@ namespace QuantLib {
         MakeOIS& withFixedLegCalendar(const Calendar& cal);
         MakeOIS& withOvernightLegCalendar(const Calendar& cal);
 
+        MakeOIS& withFixedLegConvention(BusinessDayConvention bdc);
+        MakeOIS& withOvernightLegConvention(BusinessDayConvention bdc);
+        MakeOIS& withFixedLegTerminationDateConvention(BusinessDayConvention bdc);
+        MakeOIS& withOvernightLegTerminationDateConvention(BusinessDayConvention bdc);
```

Comment:
We can also add `withConvention` and `withTerminationDateConvention` as a convenience to set the same value to both legs, like `withEndOfMonth` already does.

Reply:
And maybe `withCalendar` above, too

Reply:
Done.